walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -103,8 +103,13 @@
    * @param index Current index to be set in the current combination list.
    * @param r The size of combinations to be found.
```

Comment:
Sorry for the back and forth. Last change.
Rename this to something more relevant.

Reply:
ok